backend/src/utils/config.loader.js

```diff
@@ -25,12 +25,15 @@ export const config = {
     currentPlayersUpdateIntervalDelay: hoursToMs(
       Number(process.env.FEATURES_CURRENT_PLAYERS_UPDATE_INTERVAL_DELAY),
     ),
+    releaseDateUpdateIntervalDelay: Number(
+      process.env.FEATURES_RELEASE_DATE_UPDATE_INTERVAL_DELAY,
+    ),
     updateIntervalDelay: hoursToMs(Number(process.env.FEATURES_UPDATE_INTERVAL_DELAY)),
     iterationDelay: Number(process.env.FEATURES_ITERATION_DELAY),
   },
   runner: {
     delayFn: delay,
```

Comment:
use options as in runner in master

Reply:
Hey when you review can you leave me a message on discord explaining why we do this? Why put this into another sub property called options? And how come "expectedErrorTypes" isn't under options in runner? Just wanna understand it :)